rebasehelper/cli.py

```diff
@@ -124,6 +124,8 @@ def run():
             else:
                 main_handler.setLevel(logging.DEBUG)
             ConsoleHelper.use_colors = ConsoleHelper.should_use_colors(config)
+            logger.verbose("rebase-helper version: %s", VERSION)
```

Comment:
Unfortunately this happens before file handers are created, so the version will not be visible in log files.
I think it would make no harm to simply log the version twice, once here and second time just after file handles are created. Also, I think this information is important enough to be present in all the verbosity levels, including the default one.

Reply:
Perhaps we should log it to level INFO in CLI (to have version as the first message on stdout) and then log to VERBOSE or DEBUG in Application to ensure it gets into the logs? I think that having the version in verbose/debug log twice wouldn't do any harm.

Reply:
It won't be in the log files twice. If you mean it will be in both `debug.log` and `verbose.log`, then yes, and I don't see any reason for it not to be also in `info.log`. The version string is now more useful than ever, there is no point it hiding it.

Reply:
Hmm, I don't think that logging the version twice using the stream handler is a good approach. Also I just now realized that there is more information than this missing from the log files. Warnings ca be logged during `Application.setup` but at the that time, file handlers haven't been created. I think this would be worth fixing but it's out of the scope of this PR

Reply:
> Hmm, I don't think that logging the version twice using the stream handler is a good approach.

Well, my idea was to log only to files, but that's not so straightforward to do. Handler-level filters could be used, but that's rather hackish. Best solution would be to buffer the log messages going to stream handlers before file handlers are created, and process the buffer just after their creation. Let's revert this commit, this all deserves its own PR.